README.md

```diff
@@ -48,11 +48,6 @@ platform :ios, '11.0'
 
 1. Open the `pubspec.yaml` file located inside the app folder, and add `appcues_flutter:` under `dependencies`.
 2. Install the dependency, using `flutter pub get` from the terminal.
-3. **[⚠️ BETA ONLY]** Add the pod to your ios project's Podfile
```

Comment:
leaving other Beta terms at tops unchanged for now, since the Flutter plugin is still considered pre-release